analysis/rogueoutlaw/src/CombatLogParser.ts

```diff
@@ -9,7 +9,6 @@ import {
   EssenceOfBloodfang,
   Flagellation,
   InvigoratingShadowdust,
-  Sepsis,
```

Comment:
Why removed?

Reply:
This is an oversight. This PR started with adding onto the Sepsis module, then converting that into Typescript, then converting all of Outlaw into Typescript. I'll readd it in the following revision.